<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="viewerVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="viewerHorizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageViewer" name="imageViewer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>560</width>
            <height>560</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>560</width>
            <height>560</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="modalityVerticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="modalityTabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Calibration">
         <attribute name="title">
          <string>Calibration</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="Live">
         <attribute name="title">
          <string>Live</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>../include/ImageViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
